src/utils/DateTimeUtils.js

```diff
@@ -25,3 +25,16 @@ export const dateTimeToEpochUTCTime = (date) => {
 };
 
 export const convertDaystoSeconds = (days) => days * 86400;
+
+export const convertTo24Hour = (time12hr) => {
+  const [time, period] = time12hr?.split(" ");
+  let [hours, minutes] = time?.split(":");
+
+  if (period === "PM" && hours !== "12") {
+    hours = String(Number(hours) + 12);
+  } else if (period === "AM" && hours === "12") {
+    hours = "00";
+  }
+
+  return `${hours}:${minutes}`;
```

Comment:
can we leverage moment inbuilt method for this conversion instead of writing of our own?
const number = moment("02:00 PM", ["h:mm A"]).format("HH:mm");
console.log(number); // Output = "14.00"